### ISSUE TITLE
dockerizedVersion: "0.8.1" NOT WORKING

### ISSUE THREAD

**Issue by CoFlows** [Shard-Labs/starknet-hardhat-plugin#93]
Hi,

I am trying to update to (dockerizedVersion: "0.8.1") but its not working. This is necessary in order for "starknet.getTransaction" to work.

If I stay on 0.8.0 then "getTransaction" doesn't work but if I move to 0.8.1 then calling contracts from accounts doesn't work.

please help!

Thank you

**Comment by FabijanC** [comment]
Hi, thanks for reporting the issue.
We are aware of this and have been working to resolve it.

**Comment by FabijanC** [comment]
@CoFlows can you describe what you mean by "calling contracts from accounts doesn't work"? What exactly is the failing use case?

**Comment by CoFlows** [comment]
@FabijanC thanks for your response! I love the plugin btw. 

account0 = (await starknet.deployAccount("OpenZeppelin"));
await account0.call(mainContract, "get_time_since_deployment");

throws an error about public and private keys when using 0.8.1 but it works well on 0.8.0.

Thanks!

**Comment by FabijanC** [comment]
Is it complaining about a non-matching message signature? Can you copy the error here?

**Comment by CoFlows** [comment]
Here you go. Thank you for looking into this. To be clear, this is the error I get when running 0.8.1 and trying 

account0 = (await starknet.deployAccount("OpenZeppelin"));
await account0.call(mainContract, "get_time_since_deployment");

<img width="1730" alt="image" src="https://user-images.githubusercontent.com/25453217/164383150-534badea-3244-4405-9d40-603ba96878e7.png">

**Comment by FabijanC** [comment]
I believe this was solved with plugin v0.5.3. If so, you can close this issue.

**Comment by CoFlows** [comment]
Yes, when I tested this, the 0.5.3 had not come out yet. Thank you very much for solving this.
